src/chrome/content/rules/Phillips_Academy_Andover.xml

```diff
@@ -17,13 +6,32 @@
 	<target host="colwizlive.andover.edu" />
 	<target host="sso.andover.edu" />
 	<target host="studentreports.andover.edu" />
-	<!--target host="mail.andover.edu" /-->
 	<target host="mypassword.andover.edu" />
 	<target host="student.andover.edu" />
 	<target host="patechmasters.com" />
 	<target host="*.patechmasters.com" />
 	
+	
+	<!-- andover.edu does not respond, but should be redirected to https://www.andover.edu to prevent MitM. -->
+	<target host="andover.edu" />
+	
+	<!-- mail.andover.edu returns an error, but should be included to protect mail.andover.edu/owa -->
+	<target host="mail.andover.edu" />
+	
+	<test url="http://andover.edu" />
+	<test url="http://andover.edu/" />
+	<test url="http://mail.andover.edu" />
+	<test url="http://mail.andover.edu/" />
+	 
 	<securecookie host=".+" name=".+" />
+	
+	<!-- redirects andover.edu to https://www.andover.edu -->
+	<rule from="^http://andover\.edu/*"
```

Comment:
Assuming you intend the `*` to indicate repeats of the `/`, it should be `+` instead, to require at least one slash.